fedimint-bitcoind/src/bitcoincore_rpc.rs

```diff
@@ -47,6 +47,50 @@ pub fn from_url_to_url_auth(url: &Url) -> Result<(String, Auth)> {
         }),
         if url.username().is_empty() {
             Auth::None
+        } else if url.password().is_none() {
```

Comment:
I don't think this is the way to do this. Having no password set doesn't mean the cookie file should be used.

We should probably use cookies if certain env vars are used or something. See `fedimint-core/src/bitcoin_rpc.rs` for how it's handled. I'm not sure how exactly should it work.

Reply:
I followed the conversation on the issue. How do you feel I should implement this instead?

Reply:
IMO, some env flag to make it use auth from the cookie file would be a good start. We can figure out the other details later, I guess.

Reply:
That makes sense. So, cookie file path is set in the env to make it use the auth from the cookie file, right?

Reply:
Duly noted. Thank you.